.github-actions.yml

```diff
@@ -0,0 +1,31 @@
+name: AESD Assignment 3 Autotest
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    container:
+      image: cuaesd/aesd-autotest:24-assignment3
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+
+      - name: Install missing toolchain
+        run: |
+          apt-get update
+          apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
```

Comment:
These changes shouldn't be necessary, the toolchain is already installed in the actions runner and in the `PATH`.  Also this isn't the cross compiler you should be using/we've tested with this assignment.